streaming/java/streaming-api/src/main/java/org/ray/streaming/api/context/RuntimeContext.java

```diff
@@ -13,6 +22,22 @@
 
   Long getBatchId();
 
+  void setBatchId(long batchId);
```

Comment:
There are still some batch id code but we removed batch id before.  Can we remove `batch id`? I think in our design, batch id is not needed any more. If we need some think alike, we should use checkpointId?

Reply:
ok， i'll fix this